x-pack/plugins/reporting/export_types/csv/server/__tests__/execute_job.js

```diff
@@ -97,31 +96,36 @@ describe('CSV Execute Job', function () {
           get: configGetStub
         };
       },
-      savedObjectsClientFactory: (opts) => {
-        return new SavedObjectsClient({
-          index: '.kibana',
-          mappings: { rootType: { properties: {} } },
-          callCluster: opts.callCluster
-        });
-      },
-      uiSettingsServiceFactory: () => {
-        return {
-          get: uiSettingsGetStub
-        };
+      savedObjects: {
+        getScopedSavedObjectsClient: sinon.stub()
       },
+      uiSettingsServiceFactory: sinon.stub().returns({
+        get: uiSettingsGetStub
+      }),
       log: function () {}
     };
     mockServer.config().get.withArgs('xpack.reporting.encryptionKey').returns(encryptionKey);
     mockServer.config().get.withArgs('xpack.reporting.csv.maxSizeBytes').returns(1024 * 1000); // 1mB
     mockServer.config().get.withArgs('xpack.reporting.csv.scroll').returns({});
   });
 
+  describe('savedObjects', function () {
```

Comment:
I changed these tests when removing the `savedObjectsClientFactory` as they were testing to ensure that we were always going to use the `callWithRequest` semantics with the SavedObjectsClient and used the actual implementation of the SavedObjectsClient to ensure we always forwarded the headers, which isn't really want we want to be testing any longer.